frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/portlets/projects/ProjectPortlet.java

```diff
@@ -2746,6 +2749,92 @@ private void addLicenseToLinkedReleases(ResourceRequest request, ResourceRespons
         }
     }
 
+    private void loadSpdxLicenseInfo(ResourceRequest request, ResourceResponse response) throws IOException, PortletException {
+        final User user = UserCacheHolder.getUserFromRequest(request);
+        final String releaseId = request.getParameter(PortalConstants.RELEASE_ID);
+        final ComponentService.Iface componentClient = thriftClients.makeComponentClient();
+        final LicenseInfoService.Iface licenseInfoClient = thriftClients.makeLicenseInfoClient();
+        final JSONObject jsonResult = createJSONObject();
+        final ResourceBundle resourceBundle = ResourceBundleUtil.getBundle("content.Language", request.getLocale(), getClass());
+        final Predicate<Attachment> isISR = attachment -> AttachmentType.INITIAL_SCAN_REPORT.equals(attachment.getAttachmentType());
+
+        Set<LicenseNameWithText> licenseNameWithTexts = new HashSet<LicenseNameWithText>();
+        String attachmentContentId = "";
+        String attachmentName = "";
+        Set<String> concludedLicenseIds = new TreeSet<String>();
+        Set<String> otherLicenseNames = new TreeSet<String>();
+        AttachmentType attachmentType = AttachmentType.OTHER;
+        long totalFileCount = 0;
+        try {
+            Release release = componentClient.getReleaseById(releaseId, user);
+            Set<Attachment> attachments = CommonUtils.nullToEmptySet(release.getAttachments());
+            attachments = attachments.stream().filter(isISR).collect(Collectors.toSet());
+            if (attachments.size() == 1) {
+                Attachment attachment = attachments.iterator().next();
+                attachmentType = attachment.getAttachmentType();
+                attachmentContentId = attachment.getAttachmentContentId();
+                attachmentName = attachment.getFilename();
+                List<LicenseInfoParsingResult> licenseInfoResult = licenseInfoClient.getLicenseInfoForAttachment(release,
+                        attachmentContentId, true, user);
+                List<LicenseNameWithText> licenseWithTexts = licenseInfoResult.stream()
+                        .flatMap(result -> result.getLicenseInfo().getLicenseNamesWithTexts().stream())
```

Comment:
May be a null check on `getLicenseInfo()` and `getLicenseNamesWithTexts()`

Reply:
added bull check in `stream().filter()`